apps/test/util/netsimTestUtils.js

```diff
@@ -165,12 +165,16 @@ var fakeStorageTable = function () {
  * Fake set of storage tables for use in tests.
  */
 exports.fakeShard = function () {
+  var fakeChannel = {
+    subscribe: function () {}
```

Comment:
FYI, in the past I've tried to work around this with...
```javascript
function withUnusedParams(/* paramA, paramB */) { }
```
...but it was vetoed during review in favor of omitting the arguments.  I don't think this is captured by our style guide anywhere.  This may open the conversation again.